miasm2/jitter/codegen.py

```diff
@@ -283,6 +283,8 @@ def traverse_expr_dst(self, expr, dst2index):
                         "0")
 
         else:
+            # Spot case like Cond?X:Y
+            assert -1 not in dst2index.values()
```

Comment:
I finally convince myself that the case could not happen. Thanks for the clarification.

I've removed the commit.